fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/ShowPartitionIdCommand.java

```diff
@@ -84,13 +90,22 @@ private ShowResultSet handleShowPartitionId(ConnectContext ctx, StmtExecutor exe
                         if (partition != null) {
                             List<String> row = new ArrayList<>();
                             row.add(database.getFullName());
-                            row.add(tbl.getName());
+                            if (tbl.isTemporary()) {
+                                if (!Util.isTempTableInCurrentSession(tbl.getName())) {
+                                    continue;
+                                }
+                                row.add(Util.getTempTableDisplayName(tbl.getName()));
```

Comment:
could table provide a function to return display name directly?

Reply:
done